test/evergreen.yml

```diff
@@ -504,16 +504,28 @@ functions:
         display_name: WT Hang Analyzer Output - Execution ${execution}
         remote_file: wiredtiger/${build_variant}/${revision}/wt_hang_analyzer/wt-hang-analyzer_${task_name}_${build_id}${postfix|}.tgz
 
-  "print python stacktrace":
-    command: shell.exec
-    params:
-      working_dir: "wiredtiger/cmake_build"
-      script: |
-        set -o errexit
-        set -o verbose
-        if [ -d "WT_TEST" ]; then
-          ${python_binary|python3} ../test/evergreen/print_python_stack_trace.py -e ${python_binary|python3} -c WT_TEST -l .
-        fi
+  "dump stacktraces":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/cmake_build"
+        script: |
+          set -o errexit
+          set -o verbose
+          if [ -d "WT_TEST" ]; then
+            ${python_binary|python3} ../test/evergreen/print_stack_trace.py -e ${python_binary|python3} -c WT_TEST -l .
+          fi
+          ${python_binary|python3} ../test/evergreen/print_stack_trace.py -e ./test/format/t -c test/format -l .
```

Comment:
Can't we only use the command once, and pass the `exec_path` and `core_path` as arguments in the function from the tasks?
`${python_binary|python3} ../test/evergreen/print_stack_trace.py -e ${exec_path} -c ${core_path} -l .`